packages/blade/src/components/BaseHeaderFooter/BaseFooter.tsx

```diff
@@ -1,18 +1,24 @@
 import React from 'react';
 import { Divider } from './Divider';
 import BaseBox from '~components/Box/BaseBox';
-import { assignWithoutSideEffects } from '~utils';
+import { assignWithoutSideEffects, metaAttribute } from '~utils';
 
 type BaseFooterProps = {
   children: React.ReactNode;
+  metaDataComponentName?: string;
   showDivider?: boolean;
 };
 
-const _BaseFooter = ({ children, showDivider = true }: BaseFooterProps): React.ReactElement => {
+const _BaseFooter = ({
+  children,
+  showDivider = true,
+  metaDataComponentName,
```

Comment:
nit: or just componentName

```suggestion
  metaComponentName,
```